### ISSUE TITLE
This project is maintained?

### ISSUE THREAD

**Issue by prostomarkeloff** [awesometoolbox/ormantic#13]

**Comment by TomFaulkner** [comment]
Doesn't seem to be, there is a fork with some updates, however.

**Comment by ndavydovdev** [comment]
@TomFaulkner Maybe we will move to your fork to improve and maintain this?

**Comment by TomFaulkner** [comment]
I forked from https://github.com/MushroomMaula/ormantic and submitted a PR that was merged in. So as far as I know @MushroomMaula is the most up-to-date.

My team hasn't switched to Ormantic, though if it were maintained I would like to at some point for new services. While we could maintain it ourselves, I don't think doing it alone would be a good use of our time when sqlalchemy does what we need, albeit not ideally.

**Comment by MushroomMaula** [comment]
Yeah even though im not actively developing ormantic further, I will continue to maintain my fork.

**Comment by collerek** [comment]
Hi , I was in a need of something similar an created ormar -> you can check it out: https://github.com/collerek/ormar. I plan to both maintain and further develop it - any contributors welcome :)

**Comment by hexazero** [comment]
@collerek looks great. sorry I had to step away from this project due to time restrictions. Your work is great. Archiving this repo and putting a link to yours on this project's README.
